exec/java-exec/src/main/codegen/templates/ComparisonFunctions.java

```diff
@@ -215,6 +192,36 @@ public void eval() {
     }
   }
 
+  <#-- IS_DISTINCT_FROM function -->
+  @FunctionTemplate(names = {"is_distinct_from", "is distinct from" },
```

Comment:
I am not opposed to having a native implementation of IS [NOT] DISTINCT FROM...clearly the generated code is more compact; however adding this new functions means we would need proper functional test coverage for various data types.  Any thoughts regarding that ?